cluster-autoscaler/README.md

```diff
@@ -10,7 +10,7 @@ Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernet
 
 # FAQ/Documentation
 
-An FAQ is available [HERE](./FAQ.md).
+A FAQ is available [HERE](./FAQ.md).
```

Comment:
IMO, `An FAQ`  is grammatically accurate.

Reply:
Agree

Reply:
Duly noted. Removed from the PR. Thanks!